docs/functions.json

```diff
@@ -944,11 +944,11 @@
         "examples": [
             {
                 "template": "@(rand())",
-                "output": "0.3849275689214193274523267973563633859157562255859375"
+                "output": "0.607552015674623913099594574305228888988494873046875"
```

Comment:
bunch of these changes are just because there's now an extra call to the mocked RNG being made during setup of the session used for testing and doc generation